R/problems.R

```diff
@@ -10,7 +10,8 @@
 #'   `FALSE` (the default) the lazy data is first read completely and all
 #'   problems are returned.
 #' @return A data frame with one row for each problem and four columns:
-#'   - row,col - Row and column of problem
+#'   - row,col - Row and column number that caused the problem, referencing the
+#'   original file
```

Comment:
I suspect we should say something more generic, like "input", to account to connections and inlined/literal inputs.

Reply:
Agree